bitnami/keycloak/values.yaml

```diff
@@ -211,7 +211,11 @@ adminRealm: "master"
 ## @param production Run Keycloak in production mode. TLS configuration is required except when using proxy=edge.
 ##
 production: false
+## @param proxyHeaders Set Keycloak proxy headers
+##
+proxyHeaders: ""
 ## @param proxy reverse Proxy mode edge, reencrypt, passthrough or none
+## DEPRECATED: use proxyHeaders instead
```

Comment:
We should set the value to an empty string by default, right?

Reply:
You mean for the deprecated `proxy` value? We can, but it'll be a breaking change

Reply:
Done